Create 2021-03-03.md (#186)
solid/authorization-panel: meetings/2021-03-03.md

# 2021-03-03 Authorization Panel

https://meet.jit.si/solid-authorization

## Present
* Sarven
* Dmitri Z.
* Max Leonard
* Henry Story
* E. Pavlik
* Matthieu B.

## Agenda

* [March 10: 15:00 & 18:00 UTC: Solid/CCG meet & greet with Tim Berners Lee](https://lists.w3.org/Archives/Public/public-credentials/2021Feb/0062.html)
* [UCR Survey](https://github.com/solid/authorization-panel/blob/master/proposals/authorization-ucr/uc-survey.md) Update and actions
* [Propose access request with LDN #28](https://github.com/solid/authorization-panel/pull/28)
* [Relating ACL and ACP ontologies #148](https://github.com/solid/authorization-panel/pull/148/)
* [+ characterize axes for access relationships #179](https://github.com/solid/authorization-panel/pull/179)
* [Create wac-acp-diff-story.md #178](https://github.com/solid/authorization-panel/pull/178)

## Minutes

### [March 10: 15:00 & 18:00 UTC: Solid/CCG meet & greet with Tim Berners Lee](https://lists.w3.org/Archives/Public/public-credentials/2021Feb/0062.html)

> Dmitri: W3C Credentials CG is a large community. It incubated DID WG, VC WG and now it incubates other task forces. Solid community expressed interest in using VC, signatures and encryptions. Next Wendsday they are coming to present during our weekly call and we will present on the same day during theirs.

Reviewer: ```suggestion
Dmitri: W3C Credentials CG is a large community. It incubated DID WG, VC WG and now it incubates other task forces. Solid community expressed interest in using VC, signatures and encryptions. Next Wednesday they are coming to present during our weekly call and we will present on the same day during theirs.
```

> ...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity pased access control.

Reviewer: ```suggestion
...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity passed access control.
```
maybe "passed"? maybe "passing"? maybe something else?

Pavlik: will put it on next interop panel meeting agenda to give feedback how Access Needs can be used to request access

Sarven: should move to solid/specificatation as it is a protocol issue.

Henry: I agree this would be interesting to look at what part is a protocol. I can look at it more carefully. leave it as is for now.. for more discussion

Pavlik: discuss in data-interop for the altnerative. but okay to move to solid/specifcation 

### [Relating ACL and ACP ontologies #148](https://github.com/solid/authorization-panel/pull/148/)

Henry: after looking at it: This could be an interesting document to keep up to date. But it a way I think it would be easier to work out way towards such a document by building up simple examples comparing WAC and ACP as in the following story.

### [Create wac-acp-diff-story.md #178](https://github.com/solid/authorization-panel/pull/178)

Henry: Two problems with the story (both explained in detail in PR)
  * where does the regex expression come from, that does not seem to be in the Solid Spec
  * the Shapes restrictions are too restrictive: they don't allow acl:AuthorizesAgent to be used as the object of acl:agentClass

>   * The Shapes restrictions over constrain WAC, and so make possible extensions described in issue 184 [Limitations: Reusing Policies](https://github.com/solid/authorization-panel/issues/184) seem out of bound.

Reviewer: ```suggestion
  * The Shapes restrictions over-constrain WAC, and so make possible extensions described in issue 184 [Limitations: Reusing Policies](https://github.com/solid/authorization-panel/issues/184) seem out of bounds.
```

### [March 10: 15:00 & 18:00 UTC: Solid/CCG meet & greet with Tim Berners Lee](https://lists.w3.org/Archives/Public/public-credentials/2021Feb/0062.html)

Dmitri: W3C Credentials CG is a large community. It incubated DID WG, VC WG and now it incubates other task forces. Solid community expressed interest in using VC, signatures and encryptions. Next Wednesday they are coming to present during our weekly call and we will present on the same day during theirs.

> ...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity based access control.

Reviewer: ```suggestion
...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity-based access control.
```

Reviewer: The hint of that hyphen would have made the original error ("identity-pased") and its correction clear.

Reviewer 2: Thanks @TallTed

### [March 10: 15:00 & 18:00 UTC: Solid/CCG meet & greet with Tim Berners Lee](https://lists.w3.org/Archives/Public/public-credentials/2021Feb/0062.html)

Dmitri: W3C Credentials CG is a large community. It incubated DID WG, VC WG and now it incubates other task forces. Solid community expressed interest in using VC, signatures and encryptions. Next Wednesday they are coming to present during our weekly call and we will present on the same day during theirs.

> ...: We should decide what we want to present. I'd like us to select couple of topics. For example our identity passed access control.

Reviewer: ```suggestion
...: We should decide what we want to present. I'd like us to select a couple of topics. For example, our identity past access control.
```

Reviewer 3: I think that should be identity - "based"
